src/test/java/org/kohsuke/github/AbstractGitHubApiTestBase.java

```diff
@@ -18,7 +18,7 @@
     @Before
     public void setUp() throws Exception {
         Properties props = new Properties();
-        java.io.File f = new java.io.File(System.getProperty("user.home"), ".github.kohsuke2");
+        java.io.File f = new java.io.File(System.getProperty("user.home"), ".github");
```

Comment:
`GitHub.connect()` already looks for `~/.github`, so I need to keep the original line.